DashKit/DashPumpManager.swift

```diff
@@ -17,17 +17,21 @@ public protocol PodStatusObserver: class {
     func didUpdatePodStatus()
 }
 
-public class DashPumpManager: PumpManager {
+open class DashPumpManager: PumpManager {
```

Comment:
[nit] Consider turning this into a base class `OmnipodPumpManager` (or `PodPumpManager` or `BasePodPumpManager`) and having two subclasses `DashPumpManager` and `MockPodPumpManager` so that it is clear whether it is common functionality to all or unique functionality to one or the other. This is only a thought and entirely up to you.

Reply:
The `managerIdentifier` and `localizedTitle` would be pushed into the subclasses as well as the `PumpManager` conformance. There would be one base class initializer and the convenience initializer here would be pushed down into the subclass.

Reply:
I don't disagree with that suggestion and the rationale; it does make sense.  But I am also trying to minimize any changes to the DashPumpManager, as that is really the SUT when using the demo. If I could, I wouldn't even have a new type for the Demo PumpManager, and would just inject the mock pod comm manager into DashPumpManager via some other mechanism. But I couldn't find a way to do that cleanly.

Reply:
I like @darinkrauss 's idea.  I also consider it a [nit] and leave it up to you.